app/services/testdata/Phase1TestsInvitedStatusGenerator.scala

```diff
@@ -42,22 +43,20 @@ trait Phase1TestsInvitedStatusGenerator extends ConstructiveGenerator {
   def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier) = {
 
     val sjqTest = Phase1Test(
-      cubiksUserId = 117344,
+      cubiksUserId = scala.util.Random.nextInt(10000000),
```

Comment:
Don't we need to refer to a real cubiksUserId?
 It might be useful to create a user at that status and then try to pass the test (i.e. to verify the integration with cubiks screens is right), if the cubiks user id is ramdon, we can't.